projects/igniteui-angular/src/lib/grids/grid/column.spec.ts

```diff
@@ -1101,12 +1086,215 @@ describe('IgxGrid - Column properties #grid', () => {
+            const producedDateColumn = grid.getColumnByName('ProducedDate');
+            const orderDateColumn = grid.getColumnByName('OrderDate');
+            const receiveTimeColumn = grid.getColumnByName('ReceiveTime');
+
+            producedDateColumn.editorOptions = { dateTimeFormat: 'yyyy-MM-dd' };
+            orderDateColumn.editorOptions = { dateTimeFormat: 'yyyy--MM--dd' };
+            receiveTimeColumn.editorOptions = { dateTimeFormat: 'h-mm-ss aaaaa' };
             fix.detectChanges();
 
-            const grid = fix.componentInstance.grid;
+            producedDateColumn._cells[0].setEditMode(true)
+            fix.detectChanges();
+            tick();
+
+            let inputDebugElement = fix.debugElement.query(By.directive(IgxInputDirective));
+            let dateTimeEditor = inputDebugElement.injector.get(IgxDateTimeEditorDirective);
+            dateTimeEditor.onFocus();
+            fix.detectChanges();
+
+            let input = producedDateColumn._cells[0].nativeElement.querySelector('.igx-input-group__input');
+            expect((input as any).value).toEqual('2014-10-01');
+
+            orderDateColumn._cells[0].setEditMode(true)
+            fix.detectChanges();
+            tick();
+
+            inputDebugElement = fix.debugElement.query(By.directive(IgxInputDirective));
+            dateTimeEditor = inputDebugElement.injector.get(IgxDateTimeEditorDirective);
+            dateTimeEditor.onFocus();
+            fix.detectChanges();
+
+            input = orderDateColumn._cells[0].nativeElement.querySelector('.igx-input-group__input');
+            expect((input as any).value).toEqual('2015--10--01');
+
+            receiveTimeColumn._cells[0].setEditMode(true)
+            fix.detectChanges();
+            tick();
+
+            inputDebugElement = fix.debugElement.query(By.directive(IgxInputDirective));
+            dateTimeEditor = inputDebugElement.injector.get(IgxDateTimeEditorDirective);
+            dateTimeEditor.onFocus();
+            fix.detectChanges();
+
+            expect(dateTimeEditor.nativeElement.value).toEqual('08-37-11 a');
+        }));
+
+        it('DateTime: Use pipeArgs.format as inputFormat for cell editor if numeric and editorOptions.dateTimeFormat is unset', fakeAsync(() => {
+            const orderDateColumn = grid.getColumnByName('OrderDate');
+            const firstCell = orderDateColumn._cells[0];
+            expect(firstCell.nativeElement.innerText).toEqual('Oct 1, 2015, 11:37:22 AM');
+
+            orderDateColumn.pipeArgs = { format: 'dd-MM-yyyy hh:mm aa' };
+            fix.detectChanges();
+
+            expect(firstCell.nativeElement.innerText).toEqual('01-10-2015 11:37 AM');
+
+            firstCell.setEditMode(true);
+            fix.detectChanges();
+            tick();
+
+            let input = firstCell.nativeElement.querySelector('.igx-input-group__input');
+            expect((input as any).value).toEqual('01-10-2015 11:37 AM');
```

Comment:
Since you started cleaning up the hardcoded selector elsewhere, might as well continue to query `By.directive(IgxDateTimeEditorDirective)` in all tests. Or make the selector a constant.
Also judging by the other tests at this point the editor isn't focused yet, right? So checking the the value will give you the display instead of checking the input format. IIRC, I think you need to wait for a rAF for the focus directive to kick in (i.e. `tick(16)`), though you could as well just check the `intputFormat` prop on the editor to confirm the column passed the correct options and the rest in theory is already covered by the editor tests.